mobile_back_qti/cpp/backend_qti/qti_settings.h

```diff
@@ -19,13 +19,33 @@ limitations under the License.
 
 const std::string empty_settings = "";
 
-#include "qti_settings_sd7g1.h"
-#include "qti_settings_sd7pg2.h"
-#include "qti_settings_sd8cxg3.h"
-#include "qti_settings_sd8g1.h"
-#include "qti_settings_sd8g2.h"
-#include "qti_settings_sd8pg1.h"
-#include "qti_settings_sdm778.h"
-#include "qti_settings_sdm888.h"
+// Macro for adding quotes
+#define STRINGIFY(X) STRINGIFY2(X)
+#define STRINGIFY2(X) #X
+
+#define INCLUDE_SETTINGS(device)                             \
+  STRINGIFY(mobile_back_qti/cpp/backend_qti/settings/qti_settings_##device.pbtxt.h)
```

Comment:
this line fails the lint/c++ check.
@anhappdev any suggestions to fix this?

Reply:
Running `clang-format ` I have this:
```
#define INCLUDE_SETTINGS(device)                             \
  STRINGIFY(mobile_back_qti / cpp / backend_qti / settings / \
            qti_settings_##device.pbtxt.h)
```

Reply:
Ah, sorry. Just test run the code and it failed because of the added space.
You can ignore the format like this:
```
// clang-format off
#define INCLUDE_SETTINGS(device)                             \
  STRINGIFY(mobile_back_qti/cpp/backend_qti/settings/qti_settings_##device.pbtxt.h)
// clang-format on
```

Reply:
BTW, I got another error when building the QTI backend, so please check it:
```
mobile_back_qti/cpp/backend_qti/qti_backend_helper.h:22:10: fatal error: 'SNPE/PSNPE.h' file not found
#include "SNPE/PSNPE.h"
```